src/BridgedRoom.ts

```diff
@@ -457,6 +457,91 @@ export class BridgedRoom {
         return true;
     }
 
+    public async onSlackUserLeft(slackId: string) {
+        const ghost = await this.main.getGhostForSlack(slackId, undefined, this.slackTeamId);
+        await ghost.intent.leave(this.matrixRoomId);
+    }
+
+    public async onSlackUserJoin(slackId: string, wasInvitedBy?: string) {
```

Comment:
I could not find where this or `onSlackUserLeft` is called?

Reply:
Whoops, c&p fail.